ch.elexis.core.ui/src/ch/elexis/core/ui/actions/GlobalActions.java

```diff
@@ -177,23 +178,24 @@ public void run(){
 				
 				@Override
 				public void run(){
-					File book =
-						new File(Platform.getInstallLocation().getURL().getPath() + "elexis.pdf"); //$NON-NLS-1$
-					Program proggie = Program.findProgram(".pdf"); //$NON-NLS-1$
-					if (proggie != null) {
-						logger
-							.info("will open handbook: " + book.toString() + " using: " + proggie);
-						proggie.execute(book.toString());
-					} else {
-						logger.info("will launch handbook: " + book.toString());
-						if (Program.launch(book.toString()) == false) {
-							try {
-								logger.info("will exec handbook: " + book.toString());
-								Runtime.getRuntime().exec(book.toString());
-							} catch (Exception e) {
-								ExHandler.handle(e);
-							}
+					Desktop desktop = Desktop.getDesktop();
+					String url = "https://wiki.elexis.info";
+					
+					if(desktop != null)
+					{
+						try 
+						{
+							desktop.browse(new java.net.URI(url)); 
+						}
+						catch (Exception e)
+						{
+							logger.warn("failed to open default browser :" + e);
+							ExHandler.handle(e);
 						}
+							
+					} else {
+						
```

Comment:
Please remove empty line, be sure to use our formatter profile https://github.com/elexis/elexis-3-core/blob/master/3lexisFormatterProfile.xml in your Eclipse workspace!